### ISSUE TITLE
Question about training data

### ISSUE THREAD

**Issue by lyb18758** [yyyyyxie/textflux#5]
Thanks for sharing awesome work! After reading the paper, I had some questions about the training data：

1. The paper mentions that the input of the model is Concat([Iglyph, Iscene], axis), is this the visual input of the actual training?
2. The paper shows Fig4 about the part of the original image that needs to be edited in the model input is also masked, is this what needs to be done in the training stage?
3. We speculate that the style of the model training data is: the rendered text mask is stitched with the original image containing the mask of the area to be edited, and the mask image is stitched with the pure black image to be used as the data composition and training, I wonder if it is correct? Or, could you please provide a detailed description of the composition of the training data as well as the input and output? Thank you very much

Sincerely hope to get your  reply！

**Comment by yyyyyyxie** [comment]
Thank you for your questions. Your understanding is largely correct. Here is our process:

First, as you understood, we concatenate the glyph image (I_glyph) with the masked scene image (I_scene) to generate a wider or taller combined image, which is then concatenated with a pure mask image and the input noise along the channel dimension.

**Comment by lyb18758** [comment]
> Thank you for your questions. Your understanding is largely correct. Here is our process:
> 
> First, as you understood, we concatenate the glyph image (I_glyph) with the masked scene image (I_scene) to generate a wider or taller combined image, which is then concatenated with a pure mask image and the input noise along the channel dimension.

Thank you for your reply! 
So does that mean the glyph image (I_glyph) is connected with the mask scene image (I_scene) to form a long image, which is twice the size of the scene image? Is the mask image also twice the size of the scene image to align with the length image (i.e., half mask and half pure black image, as shown in Fig 4 of the paper)? 
And, what is the resource usage situation for full parameter fine-tuning versus LoRA fine-tuning? How much GPU memory and training time are needed for full parameter fine-tuning and LoRA fine-tuning?

**Comment by yyyyyxie** [comment]
Yes, your understanding is correct. Regarding computing resources, LoRA fine-tuning requires at least one A100(80GB) GPU, while full parameter fine-tuning needs at least two 80GB GPUs and takes about 4 days, which is also mentioned in the paper..
